src/components/Link/Link.module.scss

```diff
@@ -0,0 +1,37 @@
+.Container {
+    &.Reset {
+        padding: 0;
+        margin: 0;
+        text-decoration: none;
+        outline: none;
+        border: none;
+        background: none;
+
+        display: inline-block;
+        width: fit-content;
+
+        // @TODO: update with theme engine variable
+        color: white !important;
+        font-size: 1rem;
+        cursor: pointer;
+    }
+
+    &.Primary {
+        text-decoration: none;
+
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+
+    &.Secondary {
```

Comment:
is the ```&``` required on these nested styles?

Reply:
For this one, `.Secondary` is added to the same element as `.Container`, so need the `&`.